File: 7-kyu/Number Of Occurrences/number-of-occurrences.js
// Name
/*
  Number Of Occurrences
*/
// Url
/*
  https://www.codewars.com/kata/52829c5fe08baf7edc00122b/train/javascript
*/
// Instructions
/*
  Write a functionthat returns the number of occurrences of an element in an array.

  Examples
  var arr = [0, 1, 2, 2, 3];
  arr.numberOfOccurrences(0) === 1;
  arr.numberOfOccurrences(4) === 0;
  arr.numberOfOccurrences(2) === 2;
  arr.numberOfOccurrences("a") === 0;
*/

Array.prototype.numberOfOccurrences = function() {
  return this.filter(num => num === arguments[0]).length;
};

File: 7-kyu/Number Of Occurrences/number-of-occurrences.py
# Number Of Occurrences
# https://www.codewars.com/kata/52829c5fe08baf7edc00122b/train/python

# Write a functionthat returns the number of occurrences of an element in an array.

# Examples
# sample = [0, 1, 2, 2, 3]
# number_of_occurrences(0, sample) == 1
# number_of_occurrences(4, sample) == 0
# number_of_occurrences(2, sample) == 2
# number_of_occurrences(3, sample) == 1

def number_of_occurrences(element, sample):
    return sample.count(element)
